src/app/clusters/operational-credentials/operational-credentials.h

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <lib/core/PeerId.h>
+#include <transport/AdminPairingTable.h>
```

Comment:
Why are these includes needed here?  In fact, why is this header needed at all?  Seems to be unused....

Reply:
I use #include <transport/AdminPairingTable.h> because I used adminpairing table and I believe PeerId has some kUndefinedNodeId value.

Reply:
Use it where, though?  What includes operational-credentials.h ?

Reply:
I had used it at some point, not anymore